doc/source/tempfile.rst

```diff
@@ -0,0 +1,82 @@
+Temporary Files
+===============
+
+Ray will produce some temporary files during running.
+They are useful for logging, debugging & sharing object store with other programs.
+
+Location of Temporary Files
+---------------------------
+
+First we introduce the concept of a session of Ray.
+
+A session contains a set of processes. A session is created by executing
+``ray start`` command or call `ray.init()` in a Python script and ended by
+executiong ``ray stop`` or call `ray.shutdown()`.
+
+For each session, Ray will create a *root temporary directory* to place all its
+temporary files. The path is `/tmp/ray_{ray_pid}@{datetime}` by default.
+
+You are allowed to change the *root temporary directory* in one of these ways:
```

Comment:
So when root temporary directory is provided, then pid and datetime is not appended?

Reply:
Yes. It is not appended because users cannot control the path precisely with pid and datetime. By the way, `pid` is used to avoid inconsistent directory creation (older implementation could crash if we start several Ray processes in parallel).